client/tsung.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
```

Comment:
How are we going to pass tsung config file to the running container? Are we going to pass it when we are building the container or when we are deploying container?

Reply:
It should be passed when deploying the container as a mounted volume

Reply:
Yes, but using this yaml file, we are not passing anything when deploying.

Reply:
The user doesn't pass the file as a parameter while deploying Tsung. Instead, the path to the conf.xml file should be updated under volumes:hostPath:path.

Reply:
If the end user wants to pass different traffic scenario:
1- first discover where the load generator is going to be deployed (specified by k8s scheduler)
2- Then ssh to the node
3- create/update the tsung file under the path!

It makes the whole process of experiments complicated, we could easily pass it as configmap or environment variable.